References/DjangoData.md

```diff
@@ -0,0 +1,43 @@
+# Django Migrations (Changes to database)
+
+Django is a python based framework that helps us provide standard rest apis to our database backend in postgres. Using a framework allows us to develop solid api endpoints for our application with less time. This document is written from windows for developers of the jobhopper application. Users should be familiar with python, postgres, virtual env and have gotten the application running. This doc explains more detail about how to change the model/data/database structure.
+
+In order to keep our django application in sync with our database, we leverage Django migrations. Migrations are generally run with the command similar to this (windows):
+
+Migrations can update either the database schema (the tables/structure) or the data inside. Both types of migrations are saved in the migrations subfolder.
+
+## See Migrations and which have been applied:
+
+(venv) C:\Users\jedpi\OneDrive\Documents\GitHub\jobhopper>python manage.py showmigrations
```

Comment:
remove your shell prefix, so it's only the command. And wrap in `` so it renders as code